cli/aeon/client.go

```diff
@@ -0,0 +1,232 @@
+package aeon
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/apex/log"
+
+	"github.com/tarantool/go-prompt"
+	"github.com/tarantool/tt/cli/aeon/cmd"
+	"github.com/tarantool/tt/cli/aeon/pb"
+	"github.com/tarantool/tt/cli/connector"
+	"github.com/tarantool/tt/cli/console"
+	"github.com/tarantool/tt/cli/formatter"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// ResultType is a custom type to format output with console.Formatter interface.
+type ResultType struct {
+	data  map[string][]any
+	count int
+}
+
+// ResultError wraps pb.Error to implement console.Formatter interface.
+type ResultError struct {
+	*pb.Error
+}
```

Comment:
It is better to move the types and methods into a separate files (and test for the public API too).